cmd/assetsvc/main.go

```diff
@@ -44,18 +44,17 @@ func setupRoutes() http.Handler {
 	// Routes
 	apiv1 := r.PathPrefix(pathPrefix).Subrouter()
 	// TODO: mnelson: Seems we could use path per endpoint handling empty params? Check.
-	apiv1.Methods("GET").Path("/clusters/{cluster}/namespaces/{namespace}/charts").Queries("name", "{chartName}", "version", "{version}", "appversion", "{appversion}").Handler(WithParams(listChartsWithFilters))
```

Comment:
I decided to get rid of the .Queries, it doesn't make sense for me to have to manually define a new line for each combination of params. I'd rather handle it at the controller so that we can also manage empty values accordingly.